idaes/models/flowsheets/methanol_flowsheet.py

```diff
@@ -0,0 +1,736 @@
+                 **reaction_props.config_dict})
+
+    # mixing feed streams
+    m.fs.M101 = Mixer(
+        default={"property_package": m.fs.thermo_params_vapor,
+                 "momentum_mixing_type": MomentumMixingType.minimize,
+                 "has_phase_equilibrium": True,
+                 "inlet_list": ['H2_WGS', 'CO_WGS']})
+
+    # pre-compression
+    m.fs.C101 = Compressor(
+        default={"dynamic": False,
+                 "property_package": m.fs.thermo_params_vapor,
+                 "compressor": True,
+                 "thermodynamic_assumption": ThermodynamicAssumption.isothermal
+                 })
+
+    # pre-heating
+    m.fs.H101 = Heater(
+        default={"property_package": m.fs.thermo_params_vapor,
+                 "has_pressure_change": False,
+                 "has_phase_equilibrium": False})
+
+    # reactor
+
+    m.fs.R101 = StoichiometricReactor(
+        default={"has_heat_transfer": True,
+                 "has_heat_of_reaction": True,
+                 "has_pressure_change": False,
+                 "property_package": m.fs.thermo_params_vapor,
+                 "reaction_package": m.fs.reaction_params})
+
+    # post-expansion
+    m.fs.T101 = Turbine(
+        default={"dynamic": False,
+                 "property_package": m.fs.thermo_params_vapor})
+
+    # post-cooling
+    m.fs.H102 = Heater(
+        default={"property_package": m.fs.thermo_params_vapor,
+                 "has_pressure_change": False,
+                 "has_phase_equilibrium": False})
+
+    # product recovery
+    m.fs.F101 = Flash(
+        default={"property_package": m.fs.thermo_params_VLE,
+                 "has_heat_transfer": True,
+                 "has_pressure_change": True})
+
+    # Build the flowsheet
+    print('Unit degrees of freedom')
+    for unit in ('M101', 'C101', 'H101', 'R101', 'T101', 'H102', 'F101'):
+        if unit == 'M101':
+            spec = 14
+        else:
+            spec = 7
+        print(str(unit)+' '+str(degrees_of_freedom(getattr(m.fs, unit))-spec))
+
+    # pre-compression
+    m.fs.s02 = Arc(source=m.fs.M101.outlet, destination=m.fs.C101.inlet)
+
+    # pre-heating
+    m.fs.s03 = Arc(source=m.fs.C101.outlet, destination=m.fs.H101.inlet)
+
+    # reactor feed
+    m.fs.s04 = Arc(source=m.fs.H101.outlet, destination=m.fs.R101.inlet)
+
+    # post-expansion
+    m.fs.s05 = Arc(source=m.fs.R101.outlet, destination=m.fs.T101.inlet)
+
+    # post-cooling
+    m.fs.s06 = Arc(source=m.fs.T101.outlet, destination=m.fs.H102.inlet)
+
+    # product recovery
+    m.fs.s07 = Arc(source=m.fs.H102.outlet, destination=m.fs.F101.inlet)
+
+    # connecting unit models
+    TransformationFactory("network.expand_arcs").apply_to(m)
+
+    # Add unit and stream specifications
+    print('Total DOF: ', degrees_of_freedom(m))
+    return m
+
+
+def set_inputs(m):
+
+    #  feed streams, post WGS
+    m.fs.M101.H2_WGS.flow_mol[0].fix(637.2)  # mol/s, relative to 177 kmol/h
```

Comment:
You should include units in all these `fix` methods.

E.g. `m.fs.M101.H2_WGS.flow_mol[0].fix(177*pyo.units.kmol/pyo.units.hour)`

Also, as mentioned elsewhere, these feed conditions should be set by `Feed` blocks rather than through the `Port` of the `Mixer`.